package.json

```diff
@@ -0,0 +1,51 @@
+{
+  "name": "commercetools-adyen-integration",
+  "version": "0.0.1",
+  "description": "Integration between Commercetools and Adyen payment service provider",
+  "main": "index.js",
+  "scripts": {
+    "test": "test",
+    "start": "node ./extensions/src/init.js",
+    "lint": "./node_modules/.bin/eslint extension",
+    "init": "CTP_PROJECT_KEY=projectKey CTP_CLIENT_ID=clientId CTP_CLIENT_SECRET=clientSecret CTP_API_URL=apiUrl CTP_AUTH_URL=authUrl API_EXTENSION_BASE_URL=extensionUrl node ./extension/scripts/initResources.js"
+  },
+  "keywords": [
+    "Adyen"
+  ],
+  "repository": {
+    "type": "git",
+    "url": "git+https://github.com/commercetools/commercetools-adyen-integration.git"
+  },
+  "author": {
+    "name": "Lam Tran",
+    "email": "lam.tran@commercetools.de"
+  },
+  "engines": {
+    "node": ">=10",
+    "npm": ">=5"
+  },
+  "contributors": [],
+  "bugs": {
+    "url": "https://github.com/commercetools/commercetools-adyen-integration/issues"
+  },
+  "homepage": "https://github.com/commercetools/commercetools-adyen-integration#readme",
+  "devDependencies": {
+    "babel-eslint": "10.0.1",
+    "eslint": "5.12.0",
+    "eslint-config-airbnb": "17.1.0",
+    "eslint-config-commercetools": "6.0.0",
+    "eslint-plugin-import": "2.14.0",
+    "ngrok": "3.1.0"
+  },
+  "dependencies": {
+    "bluebird": "3.5.3",
+    "@commercetools/api-request-builder": "5.0.0",
+    "@commercetools/sdk-client": "2.0.0",
+    "@commercetools/sdk-middleware-auth": "5.0.0",
+    "@commercetools/sdk-middleware-http": "5.0.0",
+    "@commercetools/sdk-middleware-queue": "2.0.0",
+    "@commercetools/sync-actions": "3.0.0",
+    "lodash": "4.17.11",
+    "node-fetch": "2.3.0"
```

Comment:
why all versions are fixed? Is there a specific idea behind it?

Reply:
No reason. Are there reasons not to use fix versions?

Reply:
To get all versions you can use `npm ls --depth=0`.
`^` is useful when you don't want to update all dependencies one by one

Reply:
https://github.com/commercetools/commercetools-adyen-integration/pull/4/commits/536971d0e3373203fed242299f5fb23e34407de7